user_guide/business_intelligence_reports.rst

```diff
@@ -6,9 +6,8 @@ Reports
 In OroCRM, you can create reports based on the details of any entity available in the system. 
 
 With the functionality, you can get easy-to-understand adjustable reports on the amount and activity of different 
```

Comment:
edit: Using this functionality, you can get easy-to-understand adjustable reports on the amount and activity of different